business/workloads.go

```diff
@@ -1370,12 +1380,12 @@ func fetchWorkload(ctx context.Context, layer *Layer, criteria WorkloadCriteria)
 	// Resolve ReplicaSets from Deployments
 	// Resolve ReplicationControllers from DeploymentConfigs
 	// Resolve Jobs from CronJobs
-	for cname, ctype := range controllers {
-		if ctype == kubernetes.ReplicaSetType {
+	for controllerName, controllerType := range controllers {
+		if controllerType == kubernetes.ReplicaSetType {
 			found := false
 			iFound := -1
 			for i, rs := range repset {
```

Comment:
I didn't test this yet, but when debugging with the Argo Rollout installed, with two ReplicaSets belonging to it, one of the RS was discarded because it is missing in the repset variable. This is because the GetReplicaSet method only returns one of the ReplicaSets belonging to the Rollout. 

Are you seeing both RS on the Workloads List?.

Reply:
This was resolved.